FusionMCPBridge/tests/test_live_integration.py

```
@@ -0,0 +1,382 @@
+# =============================================================================
+# Smoke Test Suite
+# =============================================================================
+
+class TestSmokeTests:
+    """
+    Quick smoke tests to verify basic functionality.
+    Run these first to catch obvious issues.
+    """
+    
+    @pytest.mark.skip(reason="Uses local make_request helper with different return signature")
+    def test_bridge_responds(self, bridge_available):
```

Ruff (0.14.10)

Unused method argument: `bridge_available`

(ARG002)